### PR TITLE
Add "Powered by Cloudberry Leaflet and Mapbox" at bottom 

### PR DESCRIPTION
**Pull request by vidhya567**
Added `powered by cloudberry along with leaflet and Map Box` at the front-end bottom.

issue #246

### CODE REVIEW

**Comment by JavierJia** [comment]
@vidhya567  I rephrased the title and description a little. wish you could get the principle.

File: neo/app/views/index.scala.html [hunk 1]
```diff
@@ -10,12 +10,13 @@
     </div>
 
     <div id='how-to'>
-      <a href="https://www.youtube.com/watch?v=XwlRm0jcsU4" target="_blank" class="btn btn-info">How to use this demo</a>
+      <a href="https://www.youtube.com/watch?v=XwlRm0jcsU4" target="_blank" class="btn btn-info btn-sm">How to use this demo</a>
     </div>
-
+      <search-bar></search-bar>
```
Comment (JavierJia) [inline neo/app/views/index.scala.html]: why change the search bar to here?

File: neo/app/views/index.scala.html [hunk 2]
```diff
@@ -10,12 +10,13 @@
...
   </div>
+    <div class ="titleofweb">
```
Comment (JavierJia) [inline neo/app/views/index.scala.html]: what's purpose of this class ?

File: neo/app/views/index.scala.html [hunk 3]
```diff
@@ -61,5 +62,17 @@
     <exception-bar></exception-bar>
   </div>
 
+  <div id ='homepage'>
+    <div class="btn btn-info btn-sm">
+      <a href="http://cloudberry.ics.uci.edu/" title="A Distributed middleware by ICS,UCI" class="white"> Powered by Cloudberry</a>
+      |
+      <a href="http://leafletjs.com" title="A JS library for interactive maps" class="white">Leaflet</a>
+       ©
```
Comment (JavierJia) [inline neo/app/views/index.scala.html]: why `©`?

File: neo/public/javascripts/map/controllers.js [hunk 4]
```diff
@@ -93,6 +93,7 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
       leafletData.getMap().then(function(map) {
         $scope.map = map;
         $scope.bounds = map.getBounds();
+        map.attributionControl.setPrefix(false);
```
Comment (JavierJia) [inline neo/public/javascripts/map/controllers.js]: it worth a comment to explain the purpose of set it to `false`.

File: neo/public/stylesheets/main.css [hunk 5]
```diff
@@ -114,4 +114,41 @@ search-bar {
     position: relative;
     left: 110%;
 
-}
\ No newline at end of file
+}
+
+#homepage{
```
Comment (JavierJia) [inline neo/public/stylesheets/main.css]: remove these not used class?

Comment (vidhya567) [inline neo/public/stylesheets/main.css]: Removed ,I had those when there was footer,title and icon.

File: neo/app/views/index.scala.html (hunk 1 above)
Comment (vidhya567) [inline neo/app/views/index.scala.html]: I wanted to make the icon , title and search bar in single row.Now we have removed that too .I removed that

File: neo/app/views/index.scala.html (hunk 2 above)
Comment (vidhya567) [inline neo/app/views/index.scala.html]: It was the class which could contain icon search bar and title in a single row.Now I removed it.

File: neo/app/views/index.scala.html [hunk 6]
```diff
@@ -61,5 +61,17 @@
     <exception-bar></exception-bar>
   </div>
 
+  <div id ='homepage'>
```
Comment (JavierJia) [inline neo/app/views/index.scala.html]: `homepage` is misleading. maybe the `powerby` ?

File: neo/public/stylesheets/main.css (hunk 5 above)
Comment (JavierJia) [inline neo/public/stylesheets/main.css]: indeed, we should rename that id to a better name that specifically point to the `powerby` element.

**Comment by JavierJia** [comment]
@vidhya567 any updates?

**Pull request merged**